tools/cli.js

```diff
@@ -116,9 +119,48 @@ commandProcessor.complete = (inputs, depth) => {
 
 commandProcessor.call = (interfaceName, methodName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
+
   state.connection.callMethod(interfaceName, methodName, args, callback);
 };
 
+commandProcessor.call.complete = (inputs, depth) => {
+  if (!state.api) return [[], depth];
+
+  let iface = inputs[depth++];
+  let method = inputs[depth];
+  // there may be multiple spaces between interface and method names
+  // this function completes both of them so handle empty element ('')
+  // in between (just skip it)
+  if (method === '' && inputs[depth + 1] !== undefined) {
+    method = inputs[++depth];
+  }
+
+  let completions = complete(iface, Object.keys(state.api));
+
+  const fullInterfaceMatch = completions.some(el => el === iface);
+
+  // we should show completions if
+  // there is more than one completion and no method provided
+  if (completions.length !== 1 && method === undefined ||
```

Comment:
Can `completions` be an empty array? If yes, then this condition contradicts the comment above.